BuildLinuxAppImage.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Environment variables
+export ARCH=x86_64
+export APPIMAGE_EXTRACT_AND_RUN=1
+export QMAKE=/usr/lib/qt6/bin/qmake
+
+# Prepare the AppDir
+DESTDIR=./AppDir ninja install
+mkdir -p ./AppDir/usr/Source/Core
+cp -r ./Source/Core/DolphinQt ./AppDir/usr/Source/Core
+rm -rf ./AppDir/usr/Source/Core/DolphinQt/CMakeFiles
+rm -rf ./AppDir/usr/Source/Core/DolphinQt/dolphin-emu_autogen
+rm ./AppDir/usr/Source/Core/DolphinQt/cmake_install.cmake
+rm ./AppDir/usr/bin/dolphin-emu-nogui
+rm ./AppDir/usr/bin/dolphin-tool
+mv ./AppDir/usr/share/dolphin-emu/sys ./AppDir/usr/bin/Sys
+rm -rf ./AppDir/usr/share/dolphin-emu
+sed -i 's/env QT_QPA_PLATFORM=xcb dolphin-emu/dolphin-emu/g' ./AppDir/usr/share/applications/dolphin-emu.desktop
+
+# Prepare Tools for building the AppImage
+wget -O linuxdeploy-x86_64.AppImage https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage
+wget -O linuxdeploy-plugin-qt-x86_64.AppImage https://github.com/linuxdeploy/linuxdeploy-plugin-qt/releases/download/continuous/linuxdeploy-plugin-qt-x86_64.AppImage
+wget -O linuxdeploy-plugin-checkrt-x86_64.sh https://github.com/linuxdeploy/linuxdeploy-plugin-checkrt/releases/download/continuous/linuxdeploy-plugin-checkrt-x86_64.sh
+wget -O appimagetool-x86_64.AppImage https://github.com/AppImage/appimagetool/releases/download/continuous/appimagetool-x86_64.AppImage
```

Comment:
Since some of those files are quite sizable, does it make sense to use timestamping (`-N` or the verbose `--timestamping`) or other ways to not redownload a file when it is the same that we already have?

Probably pointless for CI if it ends up in a folder that is not kept around though, but something to consider.

Reply:
Thanks, that is a better approach. I've chnaged the script to use timestamping now. Yes, probably pointless for CI, but probably useful when using the script locally.